standard/clause_0_front_material.adoc

```diff
@@ -18,19 +18,19 @@ Given the intended use case of downloading data to a browser, there is no functi
 
-Implementation experience has shown that CoverageJSON is an effective, efficient format, friendly to web and application developers, and therefore consistent with the current https://ogcapi.ogc.org/[OGC API developments]. CoverageJSON supports the efficient transfer, from big data stores, of useable quantities of data to lightweight clients, such as browsers and mobile applications. This enables straightfoward local manipulation of the data, for example, by science researchers. Web developers often use and are familiar with JSON formats.
+Implementation experience has shown that CoverageJSON is an effective, efficient format, friendly to web and application developers, and therefore consistent with the current OGC API developments. CoverageJSON supports the efficient transfer from big data stores of useful quantities of data to lightweight clients, such as browsers and mobile applications. This enables local manipulation of the coverage data in a format familiar to, and popular with, web developers, and readily usable, for example, by science researchers.
 
 CoverageJSON can be used to encode coverages and collections of coverages. Coverage data may be gridded or non-gridded, and data values may represent continuous values (such as temperature) or discrete categories (such as land cover classes). CoverageJSON uses JSON-LD to provide interoperability with RDF and Semantic Web applications and to reduce the potential size of the payload.
 
-Relatively large datasets can be handled efficiently in a "web-friendly" way by partitioning information among several CoverageJSON documents, including a tiling mechanism. Nevertheless, CoverageJSON is not intended to be a replacement for efficient binary formats such as NetCDF, HDF or GRIB, and is not intended primarily to store or transfer very large datasets in bulk.
+Relatively large datasets can be handled efficiently in a “web-friendly” way by partitioning information among several CoverageJSON documents, including a tiling mechanism. Nevertheless, CoverageJSON is not intended to be a replacement for efficient binary formats such as NetCDF, HDF or GRIB, and is not intended primarily to store or transfer very large datasets in bulk.
 
-The simplest and most common use case is to embed all the data values of all variables in a Coverage object within the CoverageJSON document, so that it is "self-contained". A self-contained, standalone, document supports the use of very simple clients.
+Embedding the data values of all variables in a Coverage object within the CoverageJSON document is the simplest and most common use case. Such “standalone” document supports the use of very simple clients.
```

Comment:
Again, we changed this before, in order to emphasise that we are introducing use cases of increasing sophistication. We also emphasise the use of "standalone" because this is reflected in the encoding itself. I'm not sure we should change it back.

Reply:
Agreed. Unfortunately, I took a short cut of pasting most of Carl's MSWord text  in bulk, without thinking it through. I'll change most of it back.